### ISSUE TITLE
Package Request: Dwarf Fortress

### ISSUE THREAD

**Issue by johnku1** [voidlinux/void-packages#1932]
http://www.bay12games.com/dwarves/

**Comment by beefcurtains** [comment]
This is not free software.

**Comment by johnku1** [comment]
@beefcurtains Steam is included in the repo, it is not FOSS. Is this a only FOSS software repo?

**Comment by beefcurtains** [comment]
No, was just stating a fact.

**Comment by dominikh** [comment]
The sun is an incredibly hot star.

**Comment by teh-jazzman** [comment]
Nah, not really. As far as hotness goes our sun is a pretty average star :stuck_out_tongue_winking_eye:

**Comment by beefcurtains** [comment]
Bull, I don't feel heat form any other star!

**Comment by dominikh** [comment]
I'm not sure how useful a DF package would be, however. Most players will want to use some mods, at least texture mods. And as far as I am aware, those need to go in DF's directory. As such, most people would want to install DF in their home directory.

**Comment by johnku1** [comment]
Eh, I think Gentoo has a package, and not all people play with graphics mods. If they want to install the DF Lazy Newb's Pack rather then the package, it's fine.

**Comment by dominikh** [comment]
Gentoo's dwarf fortress package is in an overlay. Anyway, be our guest and submit a PR with a template for dwarf fortress and we'll merge it. I don't think any of us play unmodded df.

**Comment by Vaelatern** [comment]
So this issue is still open. Does anybody care?

**Comment by emacsomancer** [comment]
It would be nice to have. Though the points about the lazy newb's pack are valid.
